### DISCUSSION TITLE
Issue related to implementaion of a new analysis in CheckMate2.

### DISCUSSION THREAD

**Discussion by rejaafsar** [CheckMATE2/checkmate2#7] [Q&A]
Dear Developers,

 I am facing the following issues while trying to implement the ATLAS paper(arXiv:1909.09226) in CheckMate2.

1) In the ATLAS paper (arXiv:1909.09226) which I am trying to implement there are nine signal regions(SR). 
These are designed based on masses of production particles(chargino and neutralino) and mCT cut. 
                           
- SR_LM_low_mCT   
- SR_LM_medium_mCT  
- SR_LM_high_mCT
-  SR_MM_low_mCT 
- SR_MM_medium_mCT 
-  SR_MM_high_mCT 
- SR_HM_low_mCT 
- SR_HM_medium_mCT
- SR_HM_high_mCT

LM: Low mass   MM: Medium mass  HM: High mass

I am getting always "SR_MM_high_mCT " as the most sensitive SR in the ChekMate final output
even though my SLHA  input file is designed for SR_LM and SR_HM.

I have already discussed this with Dr. Jong Soo Kim and he suggested me to look at the distribution of the corresponding cut which determines these signal regions. In my case, the value of "mT" cut classifies SR_LM ,SR_MM and SR_HM regions. I found the below distribution of no of events vs mT(X_axis). 
![Screenshot from 2021-03-09 20-47-20](https://user-images.githubusercontent.com/63516155/110503138-e755dd00-8121-11eb-861d-3e6c1f3b8f70.png)

The above plot is generated using the SLHA file designed for SR_LM region. I am not able to find any clue from this.
Could you please elaborate on why this issue is happening?

2) Also, while entering the numbers for each SR in AnalysisManger correspond to the 
          
    obs: Observed number of events
    bkg: Expected no of background events
    bkg_err: Expected total error on bkg 

 " Estimate does not work for 1.4  0.6  2.0  0.0  [1.45602197786, 4.36806593357] "  message was printed on the terminal.
The screenshot is attached below.
![Screenshot_AnalysisManager](https://user-images.githubusercontent.com/63516155/110488165-3bf25b80-8114-11eb-9dd4-87a50ad61b91.png)

Dr. Kim told me to put the numbers(S95obs and S95exp) manually in var.j and ref.dat files from the CheckMate/data/analysis_info/ATLAS_13TeV directory but I did not find those numbers(S95obs & S95exp) corresponding to the above-mentioned signal regions in the atlas paper(arXiv:1909.09226) and HepData website.

Could you please let me know where should I look for those numbers?

It would be very helpful for me if you could please help me with the above issues.

With best regards,
Reja

**Comment by agagsgroove** [comment]
Hi,

1) can you compare with the cutflows given in

https://www.hepdata.net/record/ins1755298

for the different SR and benchmark points? You have to validate your code. Examples of cutflows are given at the following link,

https://checkmate.hepforge.org/AnalysesList/ATLAS_13TeV.html

2) The numbers are given in table 6 of

https://arxiv.org/pdf/1909.09226.pdf

Cheers,

Jong
